lib/test_track_rails_client/rspec_helpers.rb

```diff
@@ -2,4 +2,8 @@
 
 RSpec.configure do |config|
   config.include TestTrackRailsClient::AssignmentHelper
+
+  config.before(:each) do
+    TestTrack::FakeServer.reset!(RSpec.configuration.seed)
```

Comment:
You do need to mutate the split registry if you're stubbing a split that doesn't exist in the registry, otherwise the vary call will blow up saying that the split isn't in the registry.